playground/nuxt.config.ts

```
@@ -1,4 +1,9 @@
 export default defineNuxtConfig({
   devtools: { enabled: true },
   compatibilityDate: '2024-06-28',
+  nitro: {
+    imports: {
+      dirs
+    }
```

ESLint

[error] Missing trailing comma.

(@stylistic/comma-dangle)